contrib/Tests/Unit/ZipkinSpanConverterTest.php

```diff
@@ -2,12 +2,12 @@
 
 declare(strict_types=1);
 
-namespace unit\Exporter;
+namespace OpenTelemetry\Contrig\Tests\Unit;
```

Comment:
I think this should be `Contrib`, not `Contrig`

Reply:
done